packages/semantic-ui/src/SelectWidget/SelectWidget.tsx

```diff
@@ -80,10 +80,10 @@ export default function SelectWidget<T = any, S extends StrictRJSFSchema = RJSFS
   const _onChange = (_: SyntheticEvent<HTMLElement>, { value }: DropdownProps) =>
     onChange(enumOptionsValueForIndex<S>(value as string[], enumOptions, optEmptyVal));
   // eslint-disable-next-line no-shadow
-  const _onBlur = (_: FocusEvent<HTMLElement>, { target: { value } }: DropdownProps) =>
-    onBlur(id, enumOptionsValueForIndex<S>(value, enumOptions, optEmptyVal));
-  const _onFocus = (_: FocusEvent<HTMLElement>, { target: { value } }: DropdownProps) =>
-    onFocus(id, enumOptionsValueForIndex<S>(value, enumOptions, optEmptyVal));
+  const _onBlur = (_: FocusEvent<HTMLElement>, { target }: DropdownProps) =>
+    onBlur(id, enumOptionsValueForIndex<S>(target && target.value, enumOptions, optEmptyVal));
+  const _onFocus = (_: FocusEvent<HTMLElement>, { target }: DropdownProps) =>
+    onFocus(id, enumOptionsValueForIndex<S>(target && target.value, enumOptions, optEmptyVal));
```

Comment:
Is this the only component that has this issue? Maybe the same change needs to be made for all of the other widgets that deal with onBlur/onFocus?

Reply:
I will look into it.